CMake/kwiver-setup-tests-python.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# Script to set up environment for python nosetests
+#
+
+
+# results of tests being run will be exported to an Xunit xml file
+if (NOSE_RUNNER)
+
+  set(no_install TRUE)
+  string(TOLOWER "${CMAKE_PROJECT_NAME}" project_name)
+
+  if (WIN32)
+
+    set(kwiver_test_output_path    "${KWIVER_BINARY_DIR}/bin" )
```

Comment:
Since this is output path, it may not matter but usually on WIN32, the bin/ directory contains Debug/Release .. or whichever configuration was selected. Not sure if that matters to capture here or not.

Reply:
That's something that should be captured, and something I had overlooked, thanks!

Reply:
Having said what I said .. the problem is that, you need to use the generator expression,  $<CONFIGURATION> for visual studio which is known only at compile time not CMake config/generate time. So, it requires something like a custom_command and can be a real pain. I'm not sure how (if at all) we've addressed that in kwiver so far.